setup/setup.sh

```diff
@@ -1,9 +1,22 @@
 #!/bin/sh
 
+PRESERVECONFIG=0
+if [ -f /opt/traccar/conf/traccar.xml ]
+then
+    cp /opt/traccar/conf/traccar.xml /opt/traccar/conf/traccar.xml.saved
+    PRESERVECONFIG=1
+fi
+
 mkdir -p /opt/traccar
 cp -r * /opt/traccar
 chmod -R go+rX /opt/traccar
 
+if [ ${PRESERVECONFIG} -eq 1 ] && [ -f /opt/traccar/conf/traccar.xml.saved ]
+then
+    mv /opt/traccar/conf/traccar.xml /opt/traccar/conf/traccar.xml.new
```

Comment:
I think we don't need to save the new file. It's unlikely to change, but can cause confusion for some users.

Reply:
Removed this move to save new template